src/data/mv_polynomial.lean

```diff
@@ -1292,4 +1323,129 @@ end
 
 end equiv
 
+/-! lemmas about `pderivative v`, the partial derivative with respect to `v : σ`-/
+section pderivative
+
+variables {R : Type} [comm_ring R]
+variable {S : Type}
+
+/-- pderivative v p is the partial derivative of p with respect to v -/
+def pderivative (v : S) (p : mv_polynomial S R) : mv_polynomial S R :=
+p.sum (λ A B, monomial (erase v A + single v (A v - 1)) (B * (A v)))
+
+lemma pderivative_monomial {a : R} {v : S} {n : ℕ} :
```

Comment:
My guess is that this would also make a good `simp`-lemma. Feel free to disagree.
```suggestion
@[simp] lemma pderivative_monomial {a : R} {v : S} {n : ℕ} :
```

Reply:
Ok, let's remove it in that case. Thanks for fixing this.